src/services/db.js

```diff
@@ -0,0 +1,16 @@
+const mysql = require ('mysql2/promise');
+
+const { DB_PASSWORD, DB_USER, DB_NAME, DB_HOST, DB_PORT } = process.env;
+
+
+module.exports = async () => {
+  const connection = await mysql.createConnection({
+      host: DB_HOST,
+      user: DB_USER,
+      password: DB_PASSWORD,
+      port: DB_PORT,
+      database: DB_NAME,
+  });
+
+  return connection;
```

Comment:
The indentation is a bit inconsistent here. Most of the project uses 4 spaces bu here you are only using 2